events/wc_society_events.txt

```diff
@@ -173,3 +173,309 @@ character_event = {
+		}
+    }
+
+    option = { #stewardship
+    	name = EVTOPTB_WCCSO.10201
+    	trigger = {
+    		NOT = {
+    			has_artifact = whisphers_stewardship_1
+				has_artifact = whisphers_stewardship_2
+				has_artifact = whisphers_stewardship_3
+				has_artifact = whisphers_stewardship_4
+    		}
+    	}
+    	hidden_effect = {
+			set_character_flag = writing_whisphers_stewardship
+    		character_event = { id = WCCSO.10202 days = 90 random = 60 }
+    	}
+		ai_chance = {
+			factor = 1
+			modifier = {
+				factor = 10
+
+				has_education_stewardship_trigger = yes
+			}
+		}
+    }
+
+    option = { #intrigue
+    	name = EVTOPTC_WCCSO.10201
+    	trigger = {
+    		NOT = {
+    			has_artifact = whisphers_intrigue_1
+				has_artifact = whisphers_intrigue_2
+				has_artifact = whisphers_intrigue_3
+				has_artifact = whisphers_intrigue_4
+    		}
+    	}
+    	hidden_effect = {
+			set_character_flag = writing_whisphers_intrigue
+    		character_event = { id = WCCSO.10202 days = 90 random = 60 }
+    	}
+		ai_chance = {
+			factor = 1
+			modifier = {
+				factor = 10
+
+				has_education_intrigue_trigger = yes
+			}
+		}
+    }
+
+	option = { #learning
+    	name = EVTOPTD_WCCSO.10201
+    	trigger = {
+    		NOT = {
+    			has_artifact = whisphers_learning_1
+				has_artifact = whisphers_learning_2
+				has_artifact = whisphers_learning_3
+				has_artifact = whisphers_learning_4
+    		}
+    	}
+    	hidden_effect = {
+			set_character_flag = writing_whisphers_learning
+    		character_event = { id = WCCSO.10202 days = 90 random = 60 }
+    	}
+		ai_chance = {
+			factor = 1
+			modifier = {
+				factor = 10
+
+				has_education_learning_trigger = yes
+			}
+		}
+    }
+}
+
+character_event = {
+	id = WCCSO.10202
+	desc = {
+		trigger = { has_character_flag = writing_whisphers_martial }
+		text = EVTDESC_WCCSO.10202_martial
+	}
+	desc = {
+		trigger = { has_character_flag = writing_whisphers_stewardship }
+		text = EVTDESC_WCCSO.10202_stewardship
+	}
+	desc = {
+		trigger = { has_character_flag = writing_whisphers_intrigue }
+		text = EVTDESC_WCCSO.10202_intrigue
+	}
+	desc = {
+		trigger = { has_character_flag = writing_whisphers_learning }
+		text = EVTDESC_WCCSO.10202_learning
+	}
+	picture = GFX_evt_secluded_in_prayer
+	border = GFX_event_normal_frame_religion
+
+	is_triggered_only = yes
+
+    trigger = { can_use_society_power_write_whisphers_trigger = yes }
+
+	option = {
+		name = EVTOPTA_WCCSO.10202
+		character_event = { id = WCCSO.10203 days = 90 random = 60 }
+		if = {
+			limit = { NOT = { trait = lunatic } }
+			random = {
+				chance = 66
+				add_trait = lunatic
+			}
+		}
+	}
+}
+
+character_event = {
+	id = WCCSO.10203
+	desc = EVTDESC_WCCSO.10203
+	picture = GFX_evt_comet
+	border = GFX_event_normal_frame_religion
+
+	is_triggered_only = yes
+
+    trigger = { can_use_society_power_write_whisphers_trigger = yes }
+	
+	immediate = {
+		if = {
+			limit = {
+				AND = {
+					piety = 250
+					NOT = { piety = 500 }
+				}
```

Comment:
`AND` is not needed inside of `limit` block in this situation

Reply:
fixed